docs/api/identity-service.md

```diff
@@ -363,6 +354,11 @@ The `type` query parameter specifies the identity type to return. Accepted value
 The `type` query parameter specifies the identity type to return. Accepted values are:
 - `aziot`: Module identity.
 
+#### Request
+```json
```

Comment:
I don't think we need to send a request body for this API.

Reply:
I included it because I was getting errors when I tried to test the API without it.  Perhaps I'm doing something wrong, but even when I tried to pass no body I'd get `{"message":"EOF while parsing a value at line 1 column 0"}`.  
```bash
curl -X PUT "http://localhost:8901/identities/modules/module01?api-version=2020-09-01&type=aziot" -H "Content-Type: application/json" -H "Content-Length: 0"
```
Whereas it worked fine when with an empty body explicitly given:
```bash
curl -X PUT "http://localhost:8901/identities/modules/module01?api-version=2020-09-01&type=aziot" -H "Content-Type: application/json" --data "{}"
```

Reply:
`Route::put` takes `Self::PutBody`, so a JSON body is required. The route uses `IgnoredAny` so the body is ignored, but it's required.

We can make `Route::put` take `Option<Self::PutBody>` instead just like `post` and `delete` do, and update `make_service!` accordingly, to support optional bodies.